### DISCUSSION TITLE
Support for Hibernate 6?

### DISCUSSION THREAD

**Discussion by rayalex** [javers/javers#1236] [Q&A] [answered]
I was wondering what would the effort be for Javers to support Hibernate 6 (mainly because SpringBoot 3/JPA uses it now by default). Recently I've migrated one of my projects to SpringBoot 3, and unfortunately, Javers stopped working (as it's still pulling version 5 of Hibernate).

**Comment by bartoszwalacik** [comment]
I will check it before xmass

**Comment by bartoszwalacik** [comment, marked as answer]
you can track the progress here https://github.com/javers/javers/issues/1238

**Reply by rayalex** [reply]
Thank you!
